.gitignore

```diff
@@ -8,4 +8,5 @@ packages/docs/out
 packages/docs/pages/Dev
 .idea
 .DS_Store
-.parcel-cache
\ No newline at end of file
+.parcel-cache
+.nova
```

Comment:
Let's remove this and add this to your user gitignore config.

Reply:
Thanks! First time hearing about it